apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -4,33 +4,38 @@ import {
     ResourceType,
 } from '@coscrad/api-interfaces';
 import { MediaPlayer } from '@coscrad/media-player';
-import { Link } from 'react-router-dom';
+import { Card, CardContent } from '@mui/material';
 import { routes } from '../../../app/routes/routes';
 import { FloatSpacerDiv, SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceNavLink } from '../shared/resource-nav-link';
 import { convertMillisecondsToSeconds } from '../utils/math/';
-import './transcribed-audio-detail.thumbnail.presenter.css';
+import styles from './transcribed-audio-detail.thumbnail.presenter.module.scss';
```

Comment:
Was there a variation where these styles were defined in typescript or javascript files. I'd like to see that at some point for comparison.